src/redfish_collector/core/generator.py

```
@@ -18,10 +18,12 @@ def redfishCollector(serverAddress,username,password):
     try:
         dataRaw, dataNewSchema, modelSchemaDir = asyncio.run(dataCollector(serverAddress,username,password,templateDir,logLevel))
         dataReconstructor(dataRaw, dataNewSchema, modelSchemaDir, serverAddress,logLevel)
+        return
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)